### PR TITLE
test: Expand mobile e2e test helpers

### PR DESCRIPTION
**Pull request by dcalhoun**
## Related PRs

- https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548

## What?

Expand the mobile e2e test helpers for account for new test coverage.

## Why?

- Account for interactions with UI elements that reside beneath the virtual home
  button on mobile devices.
- Enable testing the Preformatted block type by name.

## How?

- Add a method for detecting UI elements out of a provided bounds.
- Add Preformatted block type to e2e utils.

## Testing Instructions

n/a

### Testing Instructions for Keyboard

n/a

## Screenshots or screencast 
n/a

### CODE REVIEW

File: packages/react-native-editor/__device-tests__/pages/editor-page.js [hunk 1]
```diff
@@ -385,28 +384,9 @@ class EditorPage {
 		return await this.driver.elementsByAccessibilityId( 'Document tools' );
 	}
 
-	async addNewBlock( blockName, relativePosition ) {
-		const addButton = await this.getAddBlockButton();
-
-		if ( relativePosition === 'before' ) {
```
Comment (geriux) [inline packages/react-native-editor/__device-tests__/pages/editor-page.js:391]: We used this for a test that got removed, I think it makes sense to remove it and add it back if we need that functionality. 👍

**Review by geriux — APPROVED** [review]
LGTM! Nice to see new helpers like `toggleDarkMode` 🚀 

Great work!

File: packages/react-native-editor/__device-tests__/pages/editor-page.js (hunk 1 above)
Comment (dcalhoun) [reply packages/react-native-editor/__device-tests__/pages/editor-page.js:391]: Indeed. I captured a note of this in the individual commit removing this, but forgot to hoist that to the PR description. Sorry about that.

Comment (geriux) [reply packages/react-native-editor/__device-tests__/pages/editor-page.js:391]: No problem! 🚀

**Pull request merged**